ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -1202,3 +1202,14 @@ literals:
 - thermal_energy_accumulator
 - ground_isolation_command
 - ground_temperature_sensor
+
+## New fields for Freezer (Walk in Cold room)
+- defrost_run_status
+	- ON
+	- OFF
+- defrost_temperature_sensor
+- alert_reset_command
+	- NORMAL
+	- RESET
+- fan_run_time_accumulator
```

Comment:
Which fan is this? And what kind of device will you be creating out of this? 

I think a better approach will be to put all your canonical type recommendations together and submit the necessary subfields, fields, etc., with them. Without this context, I am not able to tell what the final device should look like.

Reply:
you can create a new entity type called FRZ in the generaltypes.yaml file and a new file for canonical types called FRZ.yaml.